SampleReadme.md

```diff
@@ -0,0 +1,243 @@
+        },
+        {
+            "type": "generic",
+            "id": "364",
+            "name": "Authentication",
+            "url": "",
+            "configuration": {
+                "Azure Active Directory v2": ""
+            }
+        },
+        {
+            "type": "luis",
+            "name": "",
+            "appId": "",
+            "authoringKey": "",
+            "subscriptionKey": "",
+            "version": "0.1",
+            "region": "westus",
+            "id": "120"
+        },
+        {
+            "type": "qna",
+            "name": "",
+            "id": "85",
+            "kbId": "",
+            "subscriptionKey": "",
+            "endpointKey": "",
+            "hostname": "https://{APP NAME}.azurewebsites.net/qnamaker"
+        },
+        {
+            "type": "dispatch",
+            "serviceIds": [
+                "120",
+                "85"
+            ],
+            "name": "",
+            "appId": "",
+            "authoringKey": "",
+            "subscriptionKey": "",
+            "version": "Dispatch",
+            "region": "westus",
+            "id": "152"
+        }
+    ],
+    "padlock": "",
+    "version": "2.0"
+}
+```
+
+## Run this Sample
+
+### Visual Studio
+- Navigate to the samples folder (`botbuilder-samples/samples/csharp_dotnetcore/{SAMPLE NAME}`), and open {SAMPLE NAME}.csproj in Visual Studio.
+- Run the project (press `F5` key).
+
+### Visual Studio Code
+- Open the `botbuilder-samples/samples/csharp_dotnetcore/{SAMPLE NAME}` sample folder.
+- Bring up a terminal, navigate to the `botbuilder-samples/samples/csharp_dotnetcore/{SAMPLE NAME}` folder, and enter `dotnet run`.
+- Type `dotnet run`.
+
+## Testing the bot using Bot Framework Emulator
+[Microsoft Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+
+- Install the Bot Framework emulator from [here](https://github.com/Microsoft/BotFramework-Emulator/releases).
+
+### Connect to bot using Bot Framework Emulator **V4**
+- Launch Bot Framework Emulator.
+- From the *File* menu, select *Open Bot Configuration*.
+- Navigate to your `.bot` file.
+
+## Deploy to Azure
+
+### Using CLI Tools
+```bash
+msbot clone services -f deploymentScripts/msbotClone -n <BOT-NAME> -l <Azure-location> --subscriptionId <Azure-subscription-id>
+```
+### Deploy from Visual Studio
+- Right click on the project in viasual studio
+- Select publish
+- Follow the steps in the prompt
+### Deprovision your bot
+- To deprovision your bot using the CLI tools :
+```bash
+az group delete --name {RESOURCE GROUP NAME}
+```
+- To deprovision your bot using the Azure Portal:
+    - Search for your resource group in the Azure portal.
+    - Click on the "..." to the left side of the resource group.
+    - Select **delete** and follow the prompt.
+
+# Further Reading
+- <LINKS TO ADDITIONAL READING>

```

Comment:
Jason, Dave and Vishwac discussed this PR and we are going to break this into few additional markdown documents

# To run this sample
## Services used
Overview of services and links to services

## Create a bot file
Instructions to create a bot file

## To get set up with services
- LUIS.ai - get set up using [CLI tools](https://luis.ai) or [portal](https://luis.ai)
- QnA Maker - get set up using [CLI tools](https://qnamaker.ai) or [portal](https://qnamaker.ai)
- Dispatch - get set up using [CLI tools](tools)
- other services ...

## Run the bot
Steps to run the bot once the services are set up.

# Deploy to azure
## Create azure bot service
- Using Azure [portal](portal)
- Using the [AZ CLI](tools)

## Update .bot file with production endpoint
- Using [AZ CLI](tools)

## Deploy bot to azure
- Using [AZ CLI](tools)
- VS Code publish